firefox-ios/Client/TabManagement/TabConfigurationProvider.swift

```diff
@@ -9,14 +9,13 @@ import WebKit
 @MainActor
 class TabConfigurationProvider {
     // A WKWebViewConfiguration used for normal tabs
-    lazy var configuration: WKEngineConfiguration = {
-        return configuration(from: prefs, isPrivate: false)
-    }()
-
-    // A WKWebViewConfiguration used for private mode tabs
-    lazy var privateConfiguration: WKEngineConfiguration = {
-        return configuration(from: prefs, isPrivate: true)
-    }()
+    var configuration: WKEngineConfiguration {
+        configuration(from: prefs, isPrivate: false)
+    }
+
+    var privateConfiguration: WKEngineConfiguration {
```

Comment:
The comment was removed 🤔 
```suggestion
    // A WKWebViewConfiguration used for private mode tabs
    var privateConfiguration: WKEngineConfiguration {
```